mvc/controllers/articles.controller.js

```diff
@@ -45,4 +45,11 @@ exports.updateArticle = (req, res, next) => {
         const article = result.rows[0]
         res.status(200).send({ article });
     }).catch(next)
+}
+
+exports.deleteComment = (req, res, next) => {
+    const id = req.params.comment_id
+    removeComment(id).then(() => {
+        res.status(204).send()
```

Comment:
A nicer alternative to this is using .sendStatus(204), just so you dont need to have an empty .send() method at the end